### ISSUE TITLE
Design UX around badge metadata display in the wild when shared across the web

### ISSUE THREAD

**Issue by threeqube** [mozilla/openbadges-badges#42]
@stenington has given this a lot of thought already as can be seen in this reference ticket: https://github.com/mozilla/openbadges-badges/issues/38#issuecomment-29096975

We should design badge metadata display best practices with open badges badges as our exemplar. 

cc/ @iamjessklein

**Comment by carlacasilli** [comment]
Just curious what this means:

"We should design badge metadata display best practices with open badges badges as our exemplar."

I'm curious because I've been thinking about two different sorts of badge definitions: one aimed at potential badge earners and one aimed at potential badge consumers. They are different audiences and I'd like to suggest that we begin addressing them from their different perspectives. Then the definitions would be more along the lines of the MOOC badges than the open badges badges.

See the etherpad that shows that differentiation: https://openbadges.etherpad.mozilla.org/mooc-badges-WIP

**Comment by threeqube** [comment]
> Just curious what this means: "We should design badge metadata display best practices with open badges badges as our exemplar."

When someone shares their open badges badge on the web, say on their blog, what is the experience for when someone clicks on that badge to understand the information behind it? I'm hoping we can define that experience as best practices of badge display in the wild. Hope that clarifies. 

> They are different audiences and I'd like to suggest that we begin addressing them from their different perspectives.

This is a great point.

**Comment by iamjessklein** [comment]
:+1:

**Comment by carlacasilli** [comment]
Yes, glad that we're moving toward defining appropriate ways to display badges! :+1:
